schema.md

```diff
@@ -0,0 +1,555 @@
+# `publiccode.yml` standard
+
+This document contains an informal description of the schema of
+`publiccode.yml`.
+
+## Structure
+
+### Country-specific extensions
+
+While the standard is structured to be meaningful on an international
+level, there are additional information that can be added that makes
+sense in specific countries, such as declaring compliance with local
+laws or regulations. The provided extension mechanism is the usage
+of country-specific sections.
+
+All country-specific sections are prefixed with the two-letter ISO 639-1
+country code, followed by a dash (`-`), followed by the section name.
+For instance `it-pianotriennale` is a section declaring compliance to
+the Italian digital transformation plan.
+
+Notice that country-specific keys within international sections are
+not allowed. Countries that want to extend the format should add one
+or many country-specific sections instead.
+
+Documentation for country specific sections is maintained in separate
+files:
+
+* Italy: [Italian extensions](it-schema.md)
+
+
+## Top-level section
+
+### Key `version`
+
+* Type: string
+* Presence: mandatory
+* Example: `"0.1"`
+
+This key specifies the version to which the current `publiccode.yml`
+adheres to, for forward compatibility. Current version is `0.1`.
+
+### Key `url`
+
+* Type: string (URL)
+* Presence: mandatory
+* Example: `"https://example.com/italia/repo.git"`
+
+This string must be a URL to the repository in which the software is
+published. If the repository is available under multiple protocols,
+prefer HTTP/HTTPS URLs.
+
+Forks created for the purpose of contributing upstreams should not
+modify this file; this helps software parsing `publiccode.yml` to
+immediately skips irrelevant forks. On the contrary, a fork that is
+meant to be maintained separately from the original software should
+modify this line, to give themselves the status of a fork.
+
+### Key `upstream-url`
+
+* Type: string or array of strings
+* Presence: optional
+* Example: "https://github.com/italia/otello"
+
+This string must be one or multiple URLs of upstream repositories from
+which this fork was created. This key is meant to create a relation
+between forked repositories that could not be found out otherwise (for
+instance, some code hosting platforms record fork relations in their
+metadata).
+
+If you fork a software with the purpose of creating an independent
+version, you should consider mentioning the upstream project in this
+key.
+
+
```

Comment:
Trim superfluous whitespace and linebreaks.

Reply:
done